metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -986,8 +164,7 @@ class LookerView:
     id: LookerViewId
```

Comment:
Moving things into the lookml_conept_context.py will increase it's responsibility and lines of code

LookerView is a dataclass , the creation logic is bit complex. The creation of this dataclass involves view_upstream and view_context and other parameters as well.  The `from_looker_dict` function is actually playing a role of factory. 

I am thinking to move LookerView to existing module i.e. `looker_dataclasses.py` and have a separate dataclass factory (dataclass_factory.py) which can handle the complexity of creating data classes and it will avoid any circular dependency in future development.